### PR TITLE
Resolving #24: API call to get user info (DO NOT merge until my #22 is merged)

### PR DESCRIPTION
**Pull request by lfvarela**
Resolving #24 

API call to get information from a list of user references per request of Ian.

Example usage: 

getUsersAttributes(
      [
        firestore.collection('users').doc('kgxbnXxwNXKIupPuIrcV'),
        firestore.collection('users').doc('cfPUhjePQZlSS0XOk6st')
      ],
      ['name', 'profile_pic_url']
    );

or 

getUsersAttributes(
      [
        ref1,
        ref2
      ],
      ['name', 'profile_pic_url']
    );

returns:

Object {
  "cfPUhjePQZlSS0XOk6st": Object {
    "name": "John",
    "profile_pic_url": "",
  },
  "kgxbnXxwNXKIupPuIrcV": Object {
    "name": "Name",
    "profile_pic_url": "",
  },
}

### CODE REVIEW

File: volunta/firebase/api.js [hunk 1]
```diff
@@ -98,3 +99,32 @@ export const getAllUserInterestedEventsDocIds = async userDocId => {
   interested_refs.forEach(ref => interested.add(ref.id));
   return interested;
 };
+
+// userDocIds: array of user doc ids for whom we want to get data from
```
Comment (susannahameyer) [inline volunta/firebase/api.js]: Update comments, looks like you changed from userDocIds to userRefs in the params

File: volunta/firebase/api.js [hunk 2]
```diff
@@ -98,3 +99,32 @@ export const getAllUserInterestedEventsDocIds = async userDocId => {
...
+  // Easy way: https://medium.com/@justintulk/how-to-query-arrays-of-data-in-firebase-aa28a90181ba
+  // Robust implementation: http://bluebirdjs.com/docs/api/promise.map.html
+  let results = {};
+  Promise.map(userRefs, ref => {
+    // Promise.map awaits for returned promises as well.
+    return ref.get();
+  })
+    .then(snapshots => {
+      snapshots.forEach(snapshot => {
+        result = {};
```
Comment (susannahameyer) [inline volunta/firebase/api.js]: declare with let to avoid global var conflicts

File: volunta/firebase/api.js [hunk 3]
```diff
@@ -98,3 +99,32 @@ export const getAllUserInterestedEventsDocIds = async userDocId => {
...
+// userDocIds: array of user doc ids for whom we want to get data from
+// attributes: array of strings representing the attributes we want to get from each user. ex: ['name', 'profile_pic_url' ]
+// Uses concurrency, should be able to work for many users (more than 100)
+// Returns a map, where each key is a user id (from the ref) and the value is another map where each key is an attribute
+// specified in attributes and the value is the one that corresponds to the user.
+export const getUsersAttributes = async (userRefs, attributes) => {
+  // Easy way: https://medium.com/@justintulk/how-to-query-arrays-of-data-in-firebase-aa28a90181ba
+  // Robust implementation: http://bluebirdjs.com/docs/api/promise.map.html
+  let results = {};
+  Promise.map(userRefs, ref => {
```
Comment (susannahameyer) [inline volunta/firebase/api.js]: So nice!!

Comment (lfvarela) [reply volunta/firebase/api.js]: :)

**Pull request merged**